recipe/meta.yaml

```diff
@@ -42,6 +42,7 @@ requirements:
     - matplotlib-base
     - pandas >=0.24.1,<1.4.0
     - scikit-learn >=0.22
+    - numpy >=1.18.5
```

Comment:
```suggestion
```

We actually don't need the numpy >=1.18.5 pin, conda-build is building against the lowest supported numpy version from conda-forge (1.21) anyway 😅